pkg/api/goprobe/api.go

```diff
@@ -54,9 +54,9 @@ type ConfigResponse struct {
 // ConfigUpdateResponse is the response to a config update
 type ConfigUpdateResponse struct {
```

Comment:
@els0r Note the change in the API. Is there any place on the OSAG side where this is used (aside from the call from `gpctl`) that I need to adapt?

Reply:
I'm glad you asked. And welcome to the horrific world of keeping an OpenAPI spec up to date. Have a look at https://github.com/els0r/goProbe/tree/main/pkg/api/spec and which data structures to add (IfaceChanges) and calls to adjust.

- [ ] OpenAPI spec

As for the OSAG context: we'll just provide the OpenAPI spec to the frontend team and they run with it. So it's important we keep it 100% aligned with the code base.

I wish someone figures out OpenAPI auto-generation for go for good. One day....